pyvista/plotting/plotter.py

```diff
@@ -3475,7 +3475,7 @@ def add_mesh(
             if scalars.ndim != 2 or scalars.shape[1] < 3 or scalars.shape[1] > 4:
                 raise ValueError('RGB array must be n_points/n_cells by 3/4 in shape.')
 
-        if algo is None and not mesh.n_points:
+        if algo is None and not self.theme.allow_empty_mesh and not mesh.n_points:
             # Algorithms may initialize with an empty mesh
             raise ValueError('Empty meshes cannot be plotted. Input mesh has zero points.')
```

Comment:
Feel free to ignore this comment or reject the suggestion, I don't mean to hold anything up, but I think many PyVista users (myself included) may not know much about the theme settings or that it even exists. If I got this error (and actually intended to plot an empty mesh) it would not be clear to me that there is a setting to work around the error.
```suggestion
            raise ValueError('Empty meshes cannot be plotted. Input mesh has zero points. To allow plotting empty meshes, set `pv.global_theme.allow_empty_mesh = True`')
```

Reply:
Great suggestion!